ee/ucp/kubernetes/deploy-with-compose.md

```diff
@@ -30,27 +30,19 @@ version: '3.3'
 services:
   web:
     build: web
-    image: dockerdemos/lab-web
-    volumes:
-     - "./web/static:/static"
+    image: dockersamples/k8s-wordsmith-web
     ports:
-     - "80:80"
+     - "8080:80"
 
-  words:
+ words:
```

Comment:
add an extra space here to properly validate the YAML

Reply:
OK - I think I got it - thanks!

Reply:
@bermudezmt Maria - would you be able to help out with a couple screen shots per @stevenfollis in https://github.com/docker/docs-private/issues/791?